api/utils/error.js

```diff
@@ -0,0 +1,18 @@
+const catchAsync = (func) => {
+  return (req, res, next) => {
+    func(req, res).catch((error) => next(error));
```

Comment:
```suggestion
    func(req, res, next).catch((error) => next(error));
```
위에서 드린 코멘트에 따라 수정되어야 하는 부분입니다.